packages/devtools/CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
-## 0.0.1
+## 0.0.14 - 2019-03-15
+* Support dark mode.  Add ```&theme=dark``` at the end of URI used to open DevTools in the Chrome browser.
```

Comment:
Not that dark mode is still being polished in the change log to set user expectations correctly.

Reply:
* Dark mode, still being polished, is available.  Add ```&theme=dark``` at the end of URI used to open DevTools in the Chrome browser.